### PR TITLE
Add FPGA Addon to feed

### PR DESCRIPTION
**Pull request by zszilagy**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-custom-device-virtual-package/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adding FPGA Addon to 20.3 feed.

### Why should this Pull Request be merged?

The new repo will be added to the official Custom Devices feed.

### What testing has been done?

n/a

### CODE REVIEW

File: control [hunk 1]
```diff
@@ -8,9 +8,9 @@ Description: Adds custom devices to VeriStand.
...
+XB-AlwaysRemoves: ni-scan-engine-veristand-2019-support, ni-scan-engine-veristand-2020-support, ni-synchronization-veristand-2019-support, ni-synchronization-veristand-2020-support, ni-engine-simulation-veristand-2019-support, ni-engine-simulation-veristand-2020-support, ni-slsc12201-veristand-2019-support, ni-slsc12201-veristand-2020-support, ni-slsc-eds-veristand-2019-support, ni-slsc-eds-veristand-2020-support, ni-routing-and-faulting-veristand-2019-support, ni-routing-and-faulting-veristand-2020-support, ni-slsc-switch-veristand-2019-support, ni-slsc-switch-veristand-2020-support, ni-embedded-data-logger-veristand-2020-support, ni-routing-and-faulting-veristand-2019-labview-support, ni-routing-and-faulting-veristand-2020-labview-support, ni-slsc-switch-veristand-2019-labview-support, ni-slsc-switch-veristand-2020-labview-support, ni-routing-and-faulting-veristand-2019-labview-examples, ni-routing-and-faulting-veristand-2020-labview-examples, ni-instrument-addon-veristand-2019-support, ni-instrument-addon-veristand-2020-support, ni-fpga-addon-2019-support (>=20.2.0), ni-fpga-addon-2020-support (>=20.2.0)
```
Comment (rtzoeller) [inline control:13]: The versions should not be listed here. See other packages in this line for an example.

**Comment by rtzoeller** [comment]
> Adding FPGA Addon to 20.2 feed.

This surprised me. Are we expecting this package to ship as part of 20.2? 

FYI @Karl-G1.

File: control (hunk 1 above)
Comment (zszilagy) [reply control]: Updated as suggested!

**Comment by niphilj** [comment]
Waiting for this review to get some attention, @zszilagy

**Comment by zszilagy** [comment]
> Waiting for this review to get some attention, @zszilagy

@niphilj I resolved conflicts and updated the PR

File: control [hunk 2]
```diff
@@ -8,9 +8,9 @@ Description: Adds custom devices to VeriStand.
...
...ation-veristand-2020-support (>=20.0.0), ni-engine-simulation-veristand-2019-support (>=20.0.0), ni-engine-simulation-veristand-2020-support (>=20.0.0), ni-slsc12201-veristand-2019-support (>=20.0.0), ni-slsc12201-veristand-2020-support (>=20.0.0), ni-slsc-eds-veristand-2019-support (>=20.0.0), ni-slsc-eds-veristand-2020-support (>=20.0.0), ni-routing-and-faulting-veristand-2019-support (>=20.0.0), ni-routing-and-faulting-veristand-2020-support (>=20.0.0), ni-slsc-switch-veristand-2019-support (>=20.0.0), ni-slsc-switch-veristand-2020-support (>=20.0.0), ni-embedded-data-logger-veristand-2020-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2019-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2020-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-examples (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-examples (>= 20.0.0), ni-instrument-addon-veristand-2019-support (>=20.0.0), ni-instrument-addon-veristand-2020-support (>=20.0.0), ni-fpga-addon-2019-support (>=20.0.0), ni-fpga-addon-2020-support (>=20.0.0),
```
Comment (buckd) [inline control:11]: The package name is ni-fpga-addon-veristand-2019-support

Comment (zszilagy) [reply control]: Feedback was implemented!

**Pull request merged**